.github/workflows/ccpp.yml

```diff
@@ -23,6 +23,13 @@ jobs:
     steps:
     - name: checkout code
       uses: actions/checkout@v2
+    - name: Install Python 3
```

Comment:
Would it make sense to put the commands to install python3 into a separate script that can be provided to users, e.g. by putting it into the folder control-toolbox/ct?

Reply:
we could do that, but since python 3 is a package available from the distribution repos like gtest. I had decided not to. If you think it would be useful we could add it in.
The script would just be the following:
```
`sudo apt update`
`sudo apt install python3 python3-dev python3-numpy python3-matplotlib
```

Reply:
I moved the python deps over to install_deps let me if that is a good solution.